### DISCUSSION TITLE
Add Microsoft (EntraID) and Google Login Button

### DISCUSSION THREAD

**Discussion by nicojmb** [codeigniter4/shield#1077] [Ideas]
Please, add possibilty to login with Microsoft button or google Button, thats would we great!

**Comment by kenjis** [comment]
You can use Google OAuth with https://github.com/datamweb/shield-oauth.

**Reply by nicojmb** [reply]
Oh, great!

And for Microsoft exist something similar?

**Reply by datamweb** [reply]
You can create a custom driver for it as described below.
https://www.shield-oauth.codeigniter4.ir/add_other_oauth/
